pkg/lib/k8s/hpa.go

```diff
@@ -63,6 +64,16 @@ func HPA(spec *HPASpec) *kautoscaling.HorizontalPodAutoscaler {
 						},
```

Comment:
Would unexpected things happen if only cpu or mem are specified in the `spec`? Would it be safer to only add to the `[]kautoscaling.MetricSpec` array if the resource type is not 0?